cylc/flow/cfgspec/workflow.py

```diff
@@ -1995,6 +1973,25 @@ def upg(cfg, descr):
         ['cylc', 'mail', 'task event batch interval'],
         silent=cylc.flow.flags.cylc7_back_compat,
     )
+    u.deprecate(
+        '8.0.0',
+        ['runtime', '__MANY__', 'suite state polling'],
+        ['runtime', '__MANY__', 'workflow state polling'],
+        silent=cylc.flow.flags.cylc7_back_compat,
+        is_section=True,
+    )
+    for item in ('host', 'user'):
+        u.obsolete(
+            '8.0.0', ['runtime', '__MANY__', 'workflow state polling', item],
+        )
```

Comment:
IMO for loop obscures this code and lengthens it.

```suggestion
    u.obsolete( '8.0.0', ['runtime', '__MANY__', 'workflow state polling', 'host'])
    u.obsolete( '8.0.0', ['runtime', '__MANY__', 'workflow state polling', 'user'])
```

Reply:
(Yeah, it originally had more items!)